### PR TITLE
output bugs

### PR DESCRIPTION
**Pull request by KrazyIvan-**
fix  bug: 
initialize ciphers... (null)
listening at (null):8388

### CODE REVIEW

File: src/server.c [hunk 1]
```diff
@@ -1145,6 +1145,10 @@ int main(int argc, char **argv)
         exit(EXIT_FAILURE);
     }
 
+    if (method == 0) {
```
Comment (madeye) [inline src/server.c]: Should be NULL.

File: src/server.c [hunk 2]
```diff
@@ -1219,7 +1223,7 @@ int main(int argc, char **argv)
             FATAL("listen() error");
         }
         setnonblocking(listenfd);
-        LOGI("listening at %s:%s", host, server_port);
+        LOGI("listening at %s:%s", host?host:"", server_port);
```
Comment (clowwindy) [inline src/server.c]: "*" is better than empty.

File: src/server.c (hunk 1 above)
Comment (KrazyIvan-) [inline src/server.c]: if its NULL mean, not specify by user, so table by default. code is correct. and its show correct.
 2015-01-23 14:04:20 INFO: initialize ciphers... table
 else will show  initialize ciphers... (null)

if -m is set i.e  -m rc4
2015-01-23 14:07:51 INFO: initialize ciphers... rc4

Comment (KrazyIvan-) [inline src/server.c]: NULL == 0 same ;-)
 if ( !method )  also same

Comment (madeye) [inline src/server.c]: You should follow the coding style here.

File: src/server.c (hunk 2 above)
Comment (madeye) [inline src/server.c]: Agreed.

**Pull request merged**